packages/application-shell/src/components/custom-view-shell/custom-view-shell.tsx

```diff
@@ -44,6 +45,7 @@ type THostEventData = {
 };
 
 type TCustomViewShellProps = {
+  apolloClient?: ApolloClient<NormalizedCacheObject>;
```

Comment:
Also allow consumers to provide their custom Apollo client (same as with Custom Applications)